packages/front-end/components/Markdown/Markdown.tsx

```diff
@@ -1,11 +1,10 @@
 import { DetailedHTMLProps, FC, HTMLAttributes } from "react";
-import markdown from "markdown-it";
-import sanitizer from "markdown-it-sanitizer";
-import mark from "markdown-it-mark";
 import clsx from "clsx";
+import ReactMarkdown from "react-markdown";
+import AuthorizedImage from "../AuthorizedImage";
 import styles from "./Markdown.module.scss";
 
-const md = markdown({ html: true, linkify: true }).use(sanitizer).use(mark);
```

Comment:
Changing the renderer is a pretty big change. How thoroughly did you test this with different markdown text? Does it turn URLs into links automatically? Does it sanitize html like the previous one did? Does it use the same custom CSS styles we configured to make the output match the rest of the app styles?

Reply:
> How thoroughly did you test this with different markdown text? 
I tested on: https://markdown-it.github.io/.  The only thing the new one doesn't render that the old one did is `==mark==` as `<mark>`

> Does it turn URLs into links automatically? 
yes

> Does it sanitize html like the previous one did?
Neither the old one nor this one renders html.  The html is escaped. 

> Does it use the same custom CSS styles we configured to make the output match the rest of the app styles?
yes we surround the `ReactMarkdown` with ```<div {...props} className={clsx(className, styles.markdown)}>``` which is what the markdown was wrapped with before.  So it should all look the same as before.  I didn't notice any differences.

Reply:
Since we store the same thing in the database now, we should be able to revert the whole changeset without a problem, as long as no one locked down their bucket in the meantime.